basics/a11y.md

```diff
@@ -2,67 +2,94 @@
 
 ## Background info
```

Comment:
I feel like we should indeed note that `accessibility` is often shortened to `a11y` but I'd prefer we use the full word in our documentation whenever we can as it reads so awkwardly otherwise.